server/routes/namespaces.ts

```diff
@@ -22,14 +22,74 @@ export default function routes({ serviceCatalogueService }: Services): Router {
   router.get('/:namespaceSlug', async (req, res) => {
     const namespaceSlug = getFormattedName(req, 'namespaceSlug')
     const namespace = await serviceCatalogueService.getNamespace({ namespaceSlug })
-    const rdsInstances = namespace.rds_instance?.map(rdsInstance => rdsInstance)
 
     const displayNamespace = {
       name: namespace.name,
-      rdsInstances,
+      pingdom_check: namespace.pingdom_check,
```

Comment:
Sorry. missed these, as we're controlling this object, we use camel case for var names/attributes so the local object wants to use `pingdomCheck`, `hmppsTemplate` etc